sample/source/shared/SampleUI.cpp

```diff
@@ -126,7 +126,7 @@ namespace nc::sample
 
             if (ImGui::Button("GraphicsTest", buttonSize))
             {
-                m_ncScene->Queue(std::make_unique<GraphicsTest>(this));
+                m_ncScene->Queue(std::make_unique<GraphicsTest>(this, nc::Vector3{20.0f, 20.0f, 40.0f}));
```

Comment:
It makes more sense to me for this to be a constant defined in `GraphicsTest` (either the object or just in the cpp). `SampleUI` can't really make an informed decision here.